client/app/scripts/superdesk-search/views/search-results.html

```diff
@@ -16,24 +16,20 @@
 
             <div sd-multi-action-bar></div>
         </header>
-
-
-
-        <div class="content" sd-updown data-items="items._items" data-select="preview(item)">
-            <div class="loading-indicator" ng-show="loading" translate>loading</div>
-
-            <div sd-search-tags></div>
-
-            <div sd-shadow>
-                <ul class="{{view}}-view list-view">
-                    <li ng-repeat="item in items._items" class="list-item-view" ng-class="{active: item === selected.preview}">
-                        <div sd-media-box></div>
-                    </li>
-                </ul>
+        <div class="content" sd-scrolled data-next-action="fetchNext" data-previous-action="fetchPrevious">
+            <div sd-updown data-items="items._items" data-select="preview(item)">
+                <div class="loading-indicator" ng-show="loading" translate>loading</div>
+                <div sd-search-tags></div>
+                <div>
+                    <ul class="{{view}}-view list-view">
+                        <li ng-repeat="item in items._items" class="list-item-view" ng-class="{active: item === selected.preview}">
```

Comment:
Oh, OK then. I just mentioned it for the reference if things become slow at some point. `Track by` could help then.